src/components/dapps/modals/InputFile.vue

```diff
@@ -5,6 +5,7 @@
   >
     <div class="tw-space-y-1 tw-text-center">
       <icon-base
+        v-if="!!!$slots.default"
```

Comment:
is `!!!` common to use?

Reply:
Actually, it isn't. I exaggerated with ! :)